esphome/components/airthings_wave_plus/sensor.py

```diff
@@ -0,0 +1,99 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import sensor
+
+from esphome.const import (
+    ESP_PLATFORM_ESP32,
+    CONF_MAC_ADDRESS,
+    CONF_UPDATE_INTERVAL,
+    DEVICE_CLASS_EMPTY,
+    DEVICE_CLASS_HUMIDITY,
+    DEVICE_CLASS_RADON,
+    DEVICE_CLASS_TEMPERATURE,
+    DEVICE_CLASS_PRESSURE,
+    UNIT_PERCENT,
+    UNIT_CELSIUS,
+    UNIT_HECTOPASCAL,
+    ICON_PERCENT,
+    ICON_THERMOMETER,
+    ICON_RADIOACTIVE,
+    ICON_MOLECULE_CO2,
+    CONF_ID,
+    CONF_RADON,
+    CONF_RADON_LONG_TERM,
+    CONF_HUMIDITY,
+    CONF_TVOC,
+    CONF_CO2,
+    CONF_PRESSURE,
+    CONF_TEMPERATURE,
+    UNIT_BECQUEREL_PER_CUBIC_METER,
+    UNIT_PARTS_PER_MILLION,
+    UNIT_PARTS_PER_BILLION,
+    ICON_RADIATOR,
+    ICON_GAUGE,
+)
+
+ESP_PLATFORMS = [ESP_PLATFORM_ESP32]
+DEPENDENCIES = []
```

Comment:
```suggestion
DEPENDENCIES = ["esp32_ble_tracker"]
```

I think ESP32 ble tracker is required for this component, no?

Reply:
Not in this current implementation, but should it? The BLE initialization is performed entirely by this component.